packages/firefly/src/defineConfig.ts

```diff
@@ -18,50 +18,71 @@ import type { SwcConfig } from "@workleap/swc-configs";
 import type webpack from "webpack";
 
 export {
-    DefineBuildHostConfigOptions, DefineBuildRemoteModuleConfigOptions, DefineDevHostConfigOptions, DefineDevRemoteModuleConfigOptions, DefineHostModuleFederationPluginOptions, DefineRemoteModuleFederationPluginOptions, Features, ModuleFederationPluginOptions,
-    Router, defineRemoteModuleFederationPluginOptions
+    DefineHostModuleFederationPluginOptions, DefineRemoteModuleFederationPluginOptions, ModuleFederationPluginOptions, Router, defineRemoteModuleFederationPluginOptions
 };
 
+export type FireflyFeatures = Omit<Features, "router" | "msw">;
```

Comment:
FireflyOptionalFeatures ?

Reply:
I am not sure about this one. I mean we don't usually include "Optional" in the name of everything that is optional.